cli/spec.py

```diff
@@ -0,0 +1,244 @@
+import clg
+import ConfigParser
+import os
+import sys
+import yaml
+import yamlordereddictloader
+
+from cli import exceptions
+from cli import utils
+
+SPEC_EXTENSION = '.spec'
+LOOKUP_DICT = {
+    '__DEFAULT__': "default"
+}
+
+TRIM_PARAMS = ['default', 'required', 'requires_only']
+
+
+def IniFileType(value):
```

Comment:
Function names should be lowercase, with words separated by underscores.
(https://www.python.org/dev/peps/pep-0008/#function-names)